# Adds the provides defined by gervill (and removes non-existing ones).
# Gervill can be found in the overlays under
# openjdk/jdk/src/share/classes/com/sun/media/sound
# (This currently overrides the one from b10 because we track Gervill CVS)
--- /home/mark/src/openjdk/jdk/src/share/classes/com/sun/media/sound/services/javax.sound.sampled.spi.FormatConversionProvider	2008-04-13 01:05:30.000000000 +0200
+++ openjdk/jdk/src/share/classes/com/sun/media/sound/services/javax.sound.sampled.spi.FormatConversionProvider	2008-05-09 02:54:26.000000000 +0200
@@ -2,3 +2,4 @@
 com.sun.media.sound.UlawCodec
 com.sun.media.sound.AlawCodec
 com.sun.media.sound.PCMtoPCMCodec
+com.sun.media.sound.AudioFloatFormatConverter
